week-1-practise/level-1/9-function-output-2.js

```diff
@@ -6,9 +6,22 @@ var result = s('code', 'your');
 console.log(result);
 
 // Answer these questions:
-// - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// - What does this program do? Run it and see, or Google some of the methods used ot understand them. 
+//it will join strings and give back a new string with the sum result. In arrays, it will join arrays by order
+//and create new array with the elements of each parameter 
 // - How many parameters does the function take?
+//as many as wanted
```

Comment:
Really? Try it with three parameters and see what happens.

What happens if you pass no parameters?

Reply:
Just adding to Edward's comment, it's important that you run the programs as you're working on them to see that they do as you expect. As you move forward into more complex program's, you'll start to see the need for more in depth testing, where you'll feed, not only the expected values into functions, but also incorrect values to find out whether they'll fail gracefully.